go.mod

```diff
@@ -2,10 +2,12 @@ module github.com/OctopusDeploy/cli
 
 go 1.19
 
+//replace github.com/OctopusDeploy/go-octopusdeploy/v2 => ../go-octopusdeploy
+
 require (
 	github.com/AlecAivazis/survey/v2 v2.3.5
 	github.com/MakeNowJust/heredoc/v2 v2.0.1
-	github.com/OctopusDeploy/go-octopusdeploy/v2 v2.3.3
+	github.com/OctopusDeploy/go-octopusdeploy/v2 v2.3.4-0.20220906224003-626a6680def2
```

Comment:
I can't fix this until someone approves the other PR please
https://github.com/OctopusDeploy/go-octopusdeploy/pull/138